apollo-biz/src/test/java/com/ctrip/framework/apollo/biz/repository/AccessKeyRepositoryTest.java

```diff
@@ -0,0 +1,50 @@
+package com.ctrip.framework.apollo.biz.repository;
+
+import static org.junit.Assert.assertEquals;
+
+import com.ctrip.framework.apollo.biz.AbstractIntegrationTest;
+import com.ctrip.framework.apollo.biz.entity.AccessKey;
+import java.util.List;
+import org.junit.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+
+
+public class AccessKeyRepositoryTest extends AbstractIntegrationTest {
+
+  @Autowired
+  private AccessKeyRepository accessKeyRepository;
+
+  @Test
+  public void testSave() {
+    String appId = "someAppId";
+    String secret = "someSecret";
+
+    AccessKey accessKey = new AccessKey();
+    accessKey.setAppId(appId);
+    accessKey.setSecret(secret);
+
+    assertEquals(0, accessKeyRepository.count());
+    accessKeyRepository.save(accessKey);
+    assertEquals(1, accessKeyRepository.count());
+  }
+
+
+  @Test
+  public void testFindByAppId() {
+    String appId = "someAppId";
+    String secret = "someSecret";
+
+    AccessKey accessKey = new AccessKey();
+    accessKey.setAppId(appId);
+    accessKey.setSecret(secret);
+    accessKey.setEnabled(true);
+
+    assertEquals(0, accessKeyRepository.count());
+    accessKeyRepository.save(accessKey);
+    assertEquals(1, accessKeyRepository.count());
+
+    List<AccessKey> accessKeyList = accessKeyRepository.findByAppId(appId);
+    assertEquals(1, accessKeyList.size());
```

Comment:
It is better to add more assert statements to test the field values, such as appId and secret?

Reply:
OK